charts/dft-backend/.helmignore

```diff
@@ -1,23 +0,0 @@
-# Patterns to ignore when building packages.
```

Comment:
Why do you remove this file? It is supposed to be right here.

Reply:
checking.

Reply:
Restored.